common/src/traits.rs

```diff
@@ -41,6 +42,18 @@ pub trait NFTExt {
 		start_nft_id: NFTId,
 		amount_in_collection: u32,
 	) -> DispatchResult;
+
+	fn get_nft(id: NFTId) -> Option<NFTData<Self::AccountId, Self::NFTOffchainDataLimit>>;
```

Comment:
Can you add doc for those trait functions? Like one liners which explain what that function does